<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vvLandmarksPanel</class>
 <widget class="QWidget" name="vvLandmarksPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;CurrentImage :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QTableWidget" name="tableWidget">
     <property name="columnCount">
      <number>7</number>
     </property>
     <column>
      <property name="text">
       <string>#</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>x</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>y</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>z</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>t</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>pixel value</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Comments</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="removeButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>remove last landmark.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../vvIcons.qrc">
       <normaloff>:/common/icons/undo.png</normaloff>:/common/icons/undo.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="loadButton">
     <property name="text">
      <string>Load</string>
     </property>
     <property name="icon">
      <iconset resource="../vvIcons.qrc">
       <normaloff>:/common/icons/open.png</normaloff>:/common/icons/open.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save</string>
     </property>
     <property name="icon">
      <iconset resource="../vvIcons.qrc">
       <normaloff>:/common/icons/filesave.png</normaloff>:/common/icons/filesave.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Coordinates are in mm</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../vvIcons.qrc"/>
 </resources>
 <connections/>
</ui>
